OracleDataIntegrator/README.md

```diff
@@ -37,7 +37,7 @@ Sample Data will look like this...
         ORACLE_PWD=Welcome1
         
 The Database 12.2.0.1 container can be started from the [GitHub Location](https://github.com/oracle/docker-images/tree/master/OracleDatabase/dockerfiles/12.2.0.1). 
-Follow the instructions to create a 12.2.0.1 based enterprise edition database.
+Follow the instructions to create a 12.2.0.1 based enterprise edition database or image can be pulled from OCR or DockerStore
```

Comment:
Please provide links, i.e. [Oracle Container Registry](https://container-registry.oracle.com) and the [Docker Store](https://store.docker.com).